### ISSUE TITLE
Fatal error: Cannot redeclare rnl_autoloader()

### ISSUE THREAD

**Issue by rubengc** [rogerlos/cmb2-metatabs-options#16]
Currently if two projects have your plugin php throws a fatal error because plugin fuctions are redeclared

Updating autoloader.php with this changes the issue get fixes:
```php
<?php
/**
 * RNL AUTOLOADER
 * Looks in /code to see if a file matching the classname exists
 *
 * @param $class
 *
 * @since 1.0.0
 */
if( ! function_exists( 'rnl_autoloader' ) ) {
	function rnl_autoloader( $class ) {
		$base_dir = __DIR__;
		$class = strtolower( $class );
		$file = $base_dir . '/code/' . $class . '.php';
		if ( file_exists( $file ) ) require $file;
	}
}
```

If your are ok with this changes I can PR them or you can commit them directly

**Comment by rogerlos** [comment]
This should be fixed in 1.2. Thanks, @rubengc !

**Comment by rubengc** [comment]
Nice!
